packages/material-ui/src/Button/Button.js

```diff
@@ -13,7 +13,7 @@ export const styles = theme => ({
     ...theme.typography.button,
     lineHeight: '1.4em', // Improve readability for multiline button.
     boxSizing: 'border-box',
-    minWidth: 88,
+    minWidth: theme.spacing.unit * 8,
```

Comment:
We've moved away from using `theme.spacing.unit` in components, but setting this to 64 would be a breaking change IMHO.

Reply:
Yeah, I think you are right, as `theme.spacing.unit * 8` is being calculated to `64` already and that changes the rendering in a few component ( not sure if that's not expected )
I've changed it for now tho if we are not using `theme.spacing.unit` anymore.